tools/readme_generator/translations/fr/LC_MESSAGES/messages.po

```diff
@@ -57,8 +59,8 @@ msgid ""
 "Using the [script helpers "
 "documentation.](https://yunohost.org/packaging_apps_helpers)"
 msgstr ""
-"Utilisez la [documentation des "
-"helpers](https://yunohost.org/packaging_apps_helpers)."
+"Utilisez la [documentation des helpers.](https ://yunohost.org/"
```

Comment:
Here too

Reply:
Ah, weblate apparently loves adding this when you save without asking you :weary:

Reply:
Fixed, it was painful